config/notification/default.json

```diff
@@ -0,0 +1,59 @@
+{
+  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^2.0.0/components/context.jsonld",
+  "import": [
+    "files-scs:config/identity/handler/account-store/default.json",
+    "files-scs:config/identity/handler/adapter-factory/webid.json",
+    "files-scs:config/identity/handler/interaction/routes.json",
+    "files-scs:config/identity/handler/provider-factory/identity.json"
+  ],
+  "@graph": [
+    {
+      "comment": "The WellKnown Builder for notifications",
+      "@id": "urn:solid-server:default:NotificationWellKnownBuilder",
+      "@type": "NotificationWellKnownBuilder"
+    },
+    {
+      "comment": "Routes all Notification related requests to the relevant handlers.",
+      "@id": "urn:solid-server:default:NotificationHandler",
+      "@type": "RouterHandler",
+      "args_baseUrl": { "@id": "urn:solid-server:default:variable:baseUrl" },
+      "args_targetExtractor": { "@id": "urn:solid-server:default:TargetExtractor" },
+      "args_allowedMethods": [ "*" ],
+      "args_allowedPathNames": [ "^/notification/.*", "^/\\.well-known/openid-configuration" ],
+      "args_handler": { "@id": "urn:solid-server:default:IdentityProviderParsingHandler" }
+    },
+    {
+      "comment": "Handles IDP input parsing.",
```

Comment:
IDP stuff should very likely not be in the notifications configuration.

Reply:
This is a work in progress. I just copied over this configuration. Will clean up when it's done.